app.py

```diff
@@ -215,19 +217,35 @@ async def fulfill_offer_task(conn, offer, order_data):
 
         node_id = lease.get('resource_uuid')
         openstack_conn = get_openstack_connection(cloud=cloud_name)
+        creating_floating_ip = order_data.get('creating_floating_ip', 'False').lower() == 'true'
```

Comment:
I think the document specifies using `create_floating_ip` instead of `creating_floating_ip`; doesn't matter too much which one we use (I kinda like the first one), but it's probably worth updating to be consistent!

Reply:
Fixed!